docs/guide/dev/best_practices/README.md

```diff
@@ -103,8 +103,8 @@ Although ACF is not a security filter, leaving it in default, automatic mode sho
 
 ### Validate preview content
 
-The [Preview](https://ckeditor.com/cke4/addon/preview) plugin displays a preview of the document as it will be shown to the end user or printed. In order for the content to be displayed as closely as possible to how it looks on your page it will not be processed or secured in any special way by the plugin. This opens up possibilities for XSS attacks, so it is highly recommended to sanitize the preview content using the {@linkapi CKEDITOR.contentPreview} event. This event allows you to modify content HTML before it is displayed in the browser.
+The [Preview](https://ckeditor.com/cke4/addon/preview) plugin displays a preview of the document as it will be shown to the end user or printed. In order for the content to be displayed as closely as possible to how it looks on your page it will not be processed or secured in any special way by the plugin. This opens up possibilities for XSS attacks, so it is highly recommended to sanitize the preview content using the {@linkapi CKEDITOR.editor#event-contentPreview `contentPreview`} event. This event allows you to modify content HTML before it is displayed in the browser.
```

Comment:
I think "`contentPreview`" causes an issue when building docs:

![image](https://user-images.githubusercontent.com/1061942/73246728-9f8b2500-41af-11ea-9ce8-27772a317b5c.png)

Reply:
Have you tested it with CKEditor 4 repo switched to `t/3661` branch?

Reply:
Works fine with `t/3661`, but it means it should target `major` branch not `stable`. I will take care of retargeting then 👌